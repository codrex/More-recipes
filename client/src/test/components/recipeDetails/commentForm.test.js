/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import toJson from 'enzyme-to-json';
import { PureCommentForm }
  from '../../../components/pages/RecipeDetails/CommentForm';

const props = {
  postReview: jest.fn(),
  handleSubmit: jest.fn(),
  submitting: false,
  id: 1,
  initialize: jest.fn(),
  success: false,
  resetSuccess: jest.fn()
};

describe('CommentForm component ', () => {
  test('expected to match snapshot', () => {
    const component = shallow(<PureCommentForm {...props} />);
    component.setProps({ success: true });
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
    expect(tree).toBeInstanceOf(Object);
  });
});
