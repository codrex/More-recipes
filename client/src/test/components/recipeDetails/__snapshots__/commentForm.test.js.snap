// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentForm component  expected to match snapshot 1`] = `
<Form
  action=""
  className=""
  disabled={false}
  id="submit"
  lg={true}
  method=""
  onSubmit={[Function]}
  primary={false}
  primaryInverse={false}
  secondary={true}
  secondaryInverse={false}
  style={Object {}}
  submitBtnText="Post review"
>
  <Field
    component={[Function]}
    id="review"
    name="review"
    placeholder="Enter your review"
    row="4"
    validate={[Function]}
  />
</Form>
`;
